contracts/protocol/bases/GroupBase.sol

```diff
@@ -94,31 +94,74 @@ contract GroupBase is ProtocolBase, IBosonGroupEvents {
         condition.tokenId = _condition.tokenId;
         condition.threshold = _condition.threshold;
         condition.maxCommits = _condition.maxCommits;
+        condition.length = _condition.length;
     }
 
     /**
      * @notice Validates that condition parameters make sense.
      *
-     * Reverts if:
-     * - EvaluationMethod.None and has fields different from 0
-     * - EvaluationMethod.Threshold and token address or maxCommits is zero
-     * - EvaluationMethod.SpecificToken and token address or maxCommits is zero
+     * A invalid condition is one that fits any of the following criteria:
```

Comment:
s/A invalid/An invalid/ ... one for the future

Reply:
I fixed on https://github.com/bosonprotocol/boson-protocol-contracts/pull/656 :)